check-diff.sh

```diff
@@ -589,6 +589,25 @@ function run_phpunit_travisci {
 		cd "$INSTALL_PATH"
 
 		echo "Location: $INSTALL_PATH"
+	elif [ "$PROJECT_TYPE" == theme ]; then
+		INSTALL_PATH="$WP_CORE_DIR/src/wp-content/themes/$PROJECT_SLUG"
+
+		# Rsync the files into the right location
+		mkdir -p "$INSTALL_PATH"
+		rsync -a $(verbose_arg) --exclude .git/hooks --delete "$PROJECT_DIR/" "$INSTALL_PATH/"
```

Comment:
I wonder, why was `--exclude .git/hooks` included here and originally for the `plugin` project type. I would think that excluding all of `.git` should be what is done instead?